implementations/rust/ockam/ockam_api/src/ui/output/utils.rs

```diff
@@ -47,6 +47,15 @@ pub fn colorize_connection_status(status: ConnectionStatus) -> CString {
     }
 }
 
+pub fn indent(indent: impl Into<String>, text: impl Into<String>) -> String {
```

Comment:
I think this is not necessary. We have some formatting tools [here](https://github.com/build-trust/ockam/blob/8db3748cf20faeb29ad1dc70d60464049959b7d7/implementations/rust/ockam/ockam_api/src/ui/terminal/fmt.rs) which we use to customize the command output as needed (example [here](https://github.com/build-trust/ockam/blob/d63a5045bd400a08da4dcc57dfe9da623ca6e0b5/implementations/rust/ockam/ockam_api/src/cloud/project/project.rs#L250)).

We can land it as it is, I'll take a look once it's merged.

Reply:
Thanks. Yes I saw in your recent PRs that there was some support to avoid duplicating single and list items.